src/cascadia/TerminalSettingsEditor/Launch.cpp

```diff
@@ -17,15 +16,6 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     {
         InitializeComponent();
 
-        INITIALIZE_BINDABLE_ENUM_SETTING(FirstWindowPreference, FirstWindowPreference, FirstWindowPreference, L"Globals_FirstWindowPreference", L"Content");
-        INITIALIZE_BINDABLE_ENUM_SETTING(LaunchMode, LaunchMode, LaunchMode, L"Globals_LaunchMode", L"Content");
-        // More options were added to the JSON mapper when the enum was made into [Flags]
-        // but we want to preserve the previous set of options in the UI.
-        _LaunchModeList.RemoveAt(7); // maximizedFullscreenFocus
-        _LaunchModeList.RemoveAt(6); // fullscreenFocus
-        _LaunchModeList.RemoveAt(3); // maximizedFullscreen
-        INITIALIZE_BINDABLE_ENUM_SETTING(WindowingBehavior, WindowingMode, WindowingMode, L"Globals_WindowingBehavior", L"Content");
-
         // BODGY

```

Comment:
This check should be in the view model. If possible, it should look something like this:
- XAML markup: bind `x:Load` to view model's `IsDefTermAvailable`
- View model: call's the settings model (or heck, maybe it should even just do the work itself?) to figure out if defterm is available

If you can't bind to `x:Load`, you probably want to do this then...
- Xaml markup: set `x:Load` to false
- `OnNavigatedTo`: once we load the view model, check if defterm is available. If that's the case, `FindName()` the control

Reply:
(wrote the comment without checking the BODGY comment) yeah looks like you have to go with route 2 above hahaha

Reply:
(from discussion on Teams) We should look into whether we can just move the `IsDefaultTerminalAvailable` function into `LaunchViewModel`

Reply:
Summary from teams discussion: 
- since `ProfileViewModel` and `TerminalPage` both call this function, its probably best to leave it in `CascadiaSettings`
- there's no point having the VM expose a function that literally just calls `IsDefaultTerminalAvailable` since it is a static function, so we'll leave it in the view